From 801211b3f805aeacef8427febe3ec5dfa558a4cd Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Thu, 11 Mar 2010 11:13:31 -0300
Subject: [PATCH 6/9] move x509 file name defines to qemu-x509.h

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1268306015-11724-7-git-send-email-kraxel@redhat.com>
Patchwork-id: 7661
O-Subject: [RHEL-6 kvm PATCH v6 06/10] move x509 file name defines to qemu-x509.h
Bugzilla: 549757
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Izik Eidus <ieidus@redhat.com>
RH-Acked-by: Yonit Halperin <yhalperi@redhat.com>

Want share them with vnc and spice.

upstream: submitted: http://patchwork.ozlabs.org/patch/46225/

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 qemu-x509.h |    9 +++++++++
 vnc-tls.c   |    7 +------
 2 files changed, 10 insertions(+), 6 deletions(-)
 create mode 100644 qemu-x509.h

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-x509.h |    9 +++++++++
 vnc-tls.c   |    7 +------
 2 files changed, 10 insertions(+), 6 deletions(-)
 create mode 100644 qemu-x509.h

diff --git a/qemu-x509.h b/qemu-x509.h
new file mode 100644
index 0000000..095aec1
--- /dev/null
+++ b/qemu-x509.h
@@ -0,0 +1,9 @@
+#ifndef QEMU_X509_H
+#define QEMU_X509_H
+
+#define X509_CA_CERT_FILE "ca-cert.pem"
+#define X509_CA_CRL_FILE "ca-crl.pem"
+#define X509_SERVER_KEY_FILE "server-key.pem"
+#define X509_SERVER_CERT_FILE "server-cert.pem"
+
+#endif /* QEMU_X509_H */
diff --git a/vnc-tls.c b/vnc-tls.c
index e0a27ad..dec626c 100644
--- a/vnc-tls.c
+++ b/vnc-tls.c
@@ -24,6 +24,7 @@
  * THE SOFTWARE.
  */
 
+#include "qemu-x509.h"
 #include "vnc.h"
 #include "qemu_socket.h"
 
@@ -419,12 +420,6 @@ static int vnc_set_x509_credential(VncDisplay *vd,
 }
 
 
-#define X509_CA_CERT_FILE "ca-cert.pem"
-#define X509_CA_CRL_FILE "ca-crl.pem"
-#define X509_SERVER_KEY_FILE "server-key.pem"
-#define X509_SERVER_CERT_FILE "server-cert.pem"
-
-
 int vnc_tls_set_x509_creds_dir(VncDisplay *vd,
                                const char *certdir)
 {
-- 
1.6.3.rc4.29.g8146

